### PR TITLE
Update semantic conventions for TracingHttpModule

### PR DESCRIPTION
**Pull request by pjanotti**
__Not ready yet__ just checking test impact of current changes.

### CODE REVIEW

File: tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs [hunk 1]
```diff
@@ -123,12 +123,13 @@ private void OnBeginRequest(object sender, EventArgs eventArgs)
 
                 string host = httpRequest.Headers.Get("Host");
                 string httpMethod = httpRequest.HttpMethod.ToUpperInvariant();
-                string url = httpRequest.RawUrl.ToLowerInvariant();
+                string url = httpRequest.Url.ToString(); // Upstream uses RawUrl, ie. the part of the URL following the domain information.
```
Comment (RassK) [inline tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs:126]: Shouldn't this be also lowered?

Comment (RassK) [reply tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs:126]: isn't it going to break cardinality?

Comment (RassK) [reply tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs:126]: ... although I would expect the portal (back-end) to handle it.

Comment (pjanotti) [reply tracer/src/Datadog.Trace/AspNet/TracingHttpModule.cs:126]: Let's continue this conversation via Slack and talk with some backend folks. If needed we revisit it.

**Pull request merged**